pontoon/base/models.py

```diff
@@ -228,6 +240,15 @@ class Project(models.Model):
     # Whether this project has changed since the last sync.
     has_changed = models.BooleanField(default=False)
 
+    # Most recent translation approved or created for this project.
+    latest_translation = models.ForeignKey(
+        'Translation',
+        blank=True,
+        null=True,
+        related_name='+',
```

Comment:
Why do we need no backwards relation? I understand we'll never use it, I'm just wondering why do we want to explicitly prevent it? Optimization?